src/commands/NewCamelKameletCommand.ts

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict'
+'use strict';
```

Comment:
question:

- I am wondering, is this usega of `use strict` on the beginning of files necessary if we have `strict` option enabled by default for config? (I mean in files which are covered by eslint during compilation...)

Reply:
@djelinek In a fact it's not necessary to keep it here. If we delete it here, nothing will change. Found out that it's good to keep it here for clarity - so when anyone reads the code, immediately  knows that this code is running in strict mode. Up to your decision if we want to keep it or remove.

Reply:
ok ok, I don't have strong opinion here, I was just curious. Thanks 🙂